LICENSE

```diff
@@ -1,7 +1,7 @@
 COPYRIGHT
 =========
 
-Copyright 2020 United States Government
+Copyright 2023 United States Government
```

Comment:
Under the Apache License, it is common, and somewhat preferred, to put copyright notices in one central place in a NOTICE file at the root of the project next to the LICENSE file, rather than have it listed on each file. It makes it easier when updating the copyright dates.

Additionally... I'm not a copyright expert, but I believe the copyright date is the date of first publication, not the most recent publication. However, it is common to put a range of dates, starting from the first publication date and ending on the most recent edit, for software projects, since the date of first publication varies, and the range covers both.

I do not think it is necessary to fix that prior to merging this PR in, but I think it's something that should be fixed (along with centralizing the copyright notices with the dates) in a subsequent change. I had previously offered to submit a PR for @jwcart2 to do it myself, but I have unfortunately not had enough time. I may have time in future if nobody else gets to it first.

Reply:
These are both fair points. I have no strong opinion, and would defer to @jwcart2 and @stephensmalley as to how our copyrights should be handled.